main.py

```diff
@@ -178,7 +178,12 @@ async def list(message: Message):
 
     # Send the message and pin it
     list_message = await message.channel.send(message_to_send)
-    await list_message.pin()
+    try:
+        await list_message.pin()
+    except Forbidden:
+        print('Insufficient permission to pin tracklist. Please give me the "manage_messages" permission and try again')
```

Comment:
This one we could probably send this as a message to the channel instead of printing to stdout?

Though if we do send it as a message, the text should be modified to be more bustified.

Reply:
I think this one is better as print instead of message since it's not really a necessary feature. If say busty couldn't join the voice chat then that might be better as a message